Dockerfiles/midstream/unit_tests.py

```diff
@@ -0,0 +1,59 @@
+
+import unittest
+import os
+import subprocess
+
+
+class Testing(unittest.TestCase):
+    # Running the container with correctly set environment variables, all tasks from run_tests.py should pass
+    def test_postive(self):
+        exec_cmd = """
+        /run_tests.py
+        """
+        result = subprocess.run(exec_cmd, shell=True)
+        self.assertEqual(0, result.returncode)
```

Comment:
Could you explain why we do not check generated file? `.errormessage` ?

Reply:
Check added.

Reply:
@jsztuka I still cannot see the change.